content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Verifying image signatures
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+We continue to add support for verifying the integrity and authenticity of artefacts within Kubewarden using the Sigstore project. In this
+post, we shall focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [blog](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Sigstore signatures for images that match the name provided. If all signature validations pass or
```

Comment:
I would prefer to be closer to the policy readme, so people realize that we validate pods. How about something similar to:

> This policy validates Pods by checking their container images for signatures (that is, containers, init containers and ephemera containers in the pod). If all signature validations pass of there is no container image that matches the image name query, the Pod will be accepted.

Reply:
done in https://github.com/kubewarden/kubewarden.io/pull/87/commits/940ae906d54e4917ebb6aa9e7d1c5bc600cb66ed